internal/protocol/message.go

```diff
@@ -150,14 +152,18 @@ func (b BlockInfo) String() string {
 }
 
 type RequestMessage struct {
-	Folder string // max:64
-	Name   string // max:8192
-	Offset uint64
-	Size   uint32
+	Folder  string // max:64
+	Name    string // max:8192
+	Offset  uint64
+	Size    uint32
+	Hash    []byte // max:64
```

Comment:
You could say that. It's a safe number that's at least as large as the hash we use, and less than a gigabyte or so. :)

(We could set it to 32 and it would work, but the XDR description and ASCII art charts are generated based on this so it'll get pasted into the protocol spec, where it seems unwise to limit us to 32 bytes per definition.)